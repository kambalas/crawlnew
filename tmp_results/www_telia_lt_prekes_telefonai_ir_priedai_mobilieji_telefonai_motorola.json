[
  {
    "name": "Motorola Razr 60 Ultra 5G",
    "price": "1299",
    "error": false
  },
  {
    "name": "Motorola Razr 50 Ultra",
    "price": "699",
    "error": false
  },
  {
    "name": "Motorola Razr 50",
    "price": "478",
    "error": false
  },
  {
    "name": "Motorola Moto E14",
    "price": "109",
    "error": false
  },
  {
    "name": "Motorola G55",
    "price": "179",
    "error": false
  },
  {
    "name": "Motorola G85",
    "price": "199",
    "error": false
  },
  {
    "name": "Motorola Edge 50 Neo",
    "price": "459",
    "error": false
  }
]