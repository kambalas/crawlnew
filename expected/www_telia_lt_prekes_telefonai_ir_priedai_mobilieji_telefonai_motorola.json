[
  {
    "name": "Motorola Razr 60 Ultra 5G 512 gb žalia",
    "price": "1299",
    "error": false
  },
  {
    "name": "Motorola Razr 50 Ultra 512 gb žalia",
    "price": "1098",
    "error": false
  },
  {
    "name": "Motorola Razr 50 256 gb pilka",
    "price": "829",
    "error": false
  },
  {
    "name": "Motorola Moto E14 64 gb pilka",
    "price": "109",
    "error": false
  },
  {
    "name": "Motorola G55 256 gb pilka",
    "price": "239",
    "error": false
  },
  {
    "name": "Motorola G85 256 gb mėlyna",
    "price": "329",
    "error": false
  },
  {
    "name": "Motorola Edge 50 Neo 512 gb pilka",
    "price": "459",
    "error": false
  }
]